api/v1/observer/observer.proto

```diff
@@ -86,10 +86,26 @@ message GetFlowsRequest {
     google.protobuf.Timestamp until = 8;
 }
 
+// RelayStatus is a message sent by hubble-relay to indicate which nodes
+// were unresponsive during the request.
+message RelayStatus {
```

Comment:
Thinking: this message definition seems generic enough that it could be reused in other gRPC APIs. I'm mostly thinking about the relay API proposal in #11026. With this in mind, what about moving the definition to `api/v1/relay/relay.proto`? And then maybe rename it to simply `Status` as `relay.RelayStatus` would clutter a bit.

Reply:
Good suggestion!